extension/llm/export/partitioner_lib.py

```diff
@@ -105,6 +105,7 @@ def get_coreml_partitioner(
 
 
 def get_qnn_partitioner(
+    soc_model,
```

Comment:
```suggestion
    soc_model = SM8650,
```

Reply:
Add add a log here

Reply:
Added a log entry here.

Reply:
Providing a default could be error-prone. Users must know which target device to run the model, so to avoid issue due to silent mismatch, it's better to ask users to explicitly provide the chipset info when requesting a qnn_partitioner.

Reply:
I'm not sure how different it is comparing to use SM8650 directly in `partitioner_lib.py`. This `partitioner_lib.py` is only used in `export_llama_lib.py` and it's either we hardcoded inside the code or one level above. Neither of them is configurable by users.

Reply:
Is partitioner_lib user-facing? My understanding is that when users work on their model, they should get the partitioner matches with the target device in their script. `export_llama_lib.py` is dedicated for llama as an example, and you just told me that it won't work for SM8450. So what's the issue of hard-coded SM8650 in `export_llama_lib.py`?